docs/7-dashboard-ui/3-Zarf-deployment-ui.md

```diff
@@ -0,0 +1,115 @@
+Zarf can only read the default cluster in your Kubeconfig file, if you wish to connect to a different cluster in the kubeconfig you will need to change it to the default cluster in the terminal. See the Kubernetes documentation on [how to configure access to multiple clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+::: 
+
+Follow these steps to connect to an existing cluster:
+
+1. Be sure to have a Kubeconfig on your machine with the cluster you wish to connect to set as the default cluster.
+2. Click connect cluster button on cluster card
+3. Select cluster name in the dialog and click connect
+4. Select a ZarfInitConfig package to deploy into cluster
+5. Follow Package deployment prompts to deploy package and initialize cluster as a Zarf cluster.
+
+### Deploy K8 and Cluster with init package with Cluster
+
+If you do not have access to an existing cluster, or would simply like to spin up a new cluster. You can do that by deploying a ZarfInitConfig package and selecting the optional K3s component. 
+
+:::info
+
+This option is currently only available for linux machines or machines using a linux box. 
+
+:::
+
+Follow these steps to deploy and initialze a new cluster:
+
+1. Click deploy package button - on Deployed packages table
+2. Select an ZarfInitPackage from the list
+3. Toggle the switch for the K3s component to deploy 
+4. Complete package deployment steps to spin up and initialze your new Zarf cluster.
+
+### Deploy additional packages
+
+Once you have a cluster connected to Zarf, you can deploy additional packages into the cluster. 
+
+Steps to deploy additional packages into cluster:
+
+1. Click deploy package button on the Deployed packages table
+2. Select the package you wish to deploy from the list
+3. Complete the package deployment steps 
+
+### Additional Package Commands
+
+Once a package is deployed into the cluster, the Web UI offers additional commands that can be executed on a package. To view these commands click on the verticle elipse at the end of the table row for the package you wish to execute the command on. The Web UI currently supports these package commands.
+
+- Update: Use when you wish to update a package with a new version of the same package.
+- Remove: Use when you wish to remove a package and all of it's resources from the cluster. This cannot be undone.
+
+![Web UI deployed packages table with a context menu showing additional package commands](../.images/dashboard/Web%20UI%20-%20package%20commands.png
+
+
+## Technical Details
+
+The web UI is packaged into the Zarf binay, so you don't have to worry about additional dependencies or trying to install it yourself! The Web UI is served through your machines local browser, running as a local host, and utlizes as the backend. 
+
+Use the Zarf Deployment UI to execute the existing Zarf CLI commands. 
+- [Zarf tools Kubectl](/docs/4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_tools_kubectl.md)
```

Comment:
```suggestion
```
Kubectl can do a ton and the UI doesn't have a generic way to interface with it so would suggest dropping this line and only keeping the kubectl top one.